source/_patterns/01-atoms/icon/scss/_icon.scss

```diff
@@ -1,53 +0,0 @@
-///
-/// The printing output for icons, includes classes and font face.
-///
-@import 'icons-generated';
-@import 'icon-mixins';
-@import '~font-awesome/scss/mixins';
-@import '~font-awesome/scss/variables';
-
-// Creates the font face tag for font icons.
-@font-face {
-  font-family: 'iconfont';
-  src: url('../font/iconfont.eot?cachebust=#{random(99999)}'); /* IE9 Compat Modes */
-  src: url('../font/iconfont.eot?#iefix') format('embedded-opentype'),
-    /* IE6-IE8 */ url('../font/iconfont.woff') format('woff'),
-    /* Pretty Modern Browsers */ url('../font/iconfont.ttf') format('truetype'),
-    /* Safari, Android, iOS */ url('../font/iconfont.svg') format('svg'); /* Legacy iOS */
-  font-weight: normal;
-  font-style: normal;
-}
-
-.icon {
-  font-family: 'iconfont';
-  font-weight: normal;
-  font-style: normal;
-}
-
-@each $name, $value in (map_get($__iconfont__data, 'iconfont')) {
-  .icon--#{$name} {
-    @include icon($name);
-  }
-}
-
-// Font-Awesome Sass Examples
```

Comment:
if we are still including font-awesome in the build (i think we are? didn't see it removed in this pr) can we still include the font awesome examples in the updated demo? i know many projects still use FA regularly for ease of use as an icon solution.

also consider bumping up from 4.7 to 5.0, which gives all of their new accessibility goodies for fonts

Reply:
This brings up a larger conversation on if we continue to use the Font Awesome library as an icon font or look to move it to their SVG spritesheet that they also provide. Thoughts? Copying over icon font for Font Awesome would be faster. Not sure I would have time to write the implementation with their SVG spritesheet before starting client work next week.

Reply:
personally i am in favor of moving to 5.0 and using their default svg + js, but i'm not opposed to spritesheets for consistency's sake. that upgrade doesn't have to happen in this PR, but at the very least i would still like to see the 4.7 examples included as-is

Reply:
I added the documentation of the current FA mixin system back into the demo docs for the new atom. 423dc1d

I agree that there is room for further development in upgrading FA in a future branch. The SVG + JS approach looks nice.